packages/backend/src/dbt/profiles.ts

```diff
@@ -174,6 +174,24 @@ const credentialsTarget = (
                     [envVar('token')]: credentials.personalAccessToken,
                 },
             };
+        case WarehouseTypes.ATHENA:
+            return {
+                target: {
+                    type: WarehouseTypes.ATHENA,
+                    // all dbt config
+                    work_group: credentials.workgroup,
+                    region_name: credentials.awsRegion,
+                    s3_staging_dir: credentials.outputLocation,
+                    schema: credentials.schema,
+                    database: credentials.database,
+                    aws_access_key_id: credentials.awsAccessKeyId,
+                    aws_secret_access_key: credentials.awsSecretKey,
```

Comment:
can this be optional? for example if I run Lightdash in a self hosted environments (e.g. ECS) I would like to leverage the underlying role used by ECS to authenticate, without hard-coding credentials anywhere, simply because any call done by container are already authenticated.

Reply:
They are, in fact that's how i was testing the build. The only thing needed is `AWS_SDK_LOAD_CONFIG=true` as environment variable.

Reply:
Thanks for the clarification. So if I get right `AWS_SDK_LOAD_CONFIG ` should be enough when running from ECS or EKS/pods with authentication via roles right?

Reply:
Yup, I've added this to `.env` file, but it should definitely be mentioned in the docs.